### PR TITLE
Add React Js & MDBootstrap

### PR DESCRIPTION
**Pull request by mbouadeus**
Added react js and mdbootstrap to project. Setup schedule activity page as well.

### CODE REVIEW

File: src/main/webapp/public/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  </head>
+  <body>
+    <noscript>
+      You need to enable JavaScript to run this app.
+    </noscript>
+    <div id="root"></div>
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
```
Comment (bhargaviks) [inline src/main/webapp/public/index.html:32]: was this comment intended? or do you want to retype it to fit our website. Like say, "This div is used in our website to ..... and has .... "

File: src/main/webapp/src/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import "bootstrap-css-only/css/bootstrap.min.css";
+import "mdbreact/dist/css/mdb.css";
+import "./index.css";
+import App from "./App";
+
+import registerServiceWorker from './registerServiceWorker';
+
+ReactDOM.render( <App /> , document.getElementById('root'));
+
+registerServiceWorker();
```
Comment (bhargaviks) [inline src/main/webapp/src/index.js:13]: Please add a new line.

File: src/main/webapp/src/pages/ScrollbarPage.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+/* custom colors  */
+.scrollbar-primary::-webkit-scrollbar {
+  background-color: #F5F5F5;
+}
+
+.scrollbar-primary::-webkit-scrollbar-thumb {
+  border-radius: 10px;
+  -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.1);
+  background-color: #4285F4;
+}
```
Comment (bhargaviks) [inline src/main/webapp/src/pages/ScrollbarPage.css:25]: add a new line at EOF

File: src/main/webapp/src/pages/Masonry.css [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+    .masonry-with-flex div:nth-child(34) {
+      height: 109px;
+      line-height: 109px; }
+    .masonry-with-flex div:nth-child(35) {
+      height: 293px;
+      line-height: 293px; }
+    .masonry-with-flex div:nth-child(36) {
+      height: 169px;
+      line-height: 169px; }
+
```
Comment (bhargaviks) [inline src/main/webapp/src/pages/Masonry.css:330]: add a new line at EOF. looks like there is an empty line, but remove spaces.

File: src/main/webapp/public/index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
```
Comment (bhargaviks) [inline src/main/webapp/public/index.html:34]: was this comment intended? or do you want to retype it to fit our website. Like say, "This div is used in our website to ..... and has .... "

**Review by bhargaviks — CHANGES_REQUESTED** [review]

File: src/main/webapp/src/index.js (hunk 2 above)
Comment (mbouadeus) [reply src/main/webapp/src/index.js:13]: Will add it in the next pull request.

File: src/main/webapp/src/pages/ScrollbarPage.css (hunk 3 above)
Comment (mbouadeus) [reply src/main/webapp/src/pages/ScrollbarPage.css:25]: Will add it in the next pull request.

File: src/main/webapp/src/pages/Masonry.css (hunk 4 above)
Comment (mbouadeus) [reply src/main/webapp/src/pages/Masonry.css:330]: Will add it in the next pull request.

File: src/main/webapp/public/index.html (hunk 5 above)
Comment (mbouadeus) [reply src/main/webapp/public/index.html:34]: The comment came along with the react starter files. It's just meant to explain how react will use the div tag.

Comment (trinitydonohugh) [reply src/main/webapp/public/index.html:34]: Can we delete the comments that came with the starter file and add comments personalized to our page? :)

File: src/main/webapp/src/App-clear.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+import logo from "./logo.png";
+
+class App extends Component {
+  render() {
+    return (
+      <MDBContainer>
+        <MDBRow center style={{ height: "100vh" }}>
+          <MDBCol middle="true" sm="8" className="text-center">
+            <img src={logo} alt="logo" style={{ width: "10rem" }} />
+            <h1>Welcome to Your MDB React App</h1>
```
Comment (trinitydonohugh) [inline src/main/webapp/src/App-clear.js:13]: Can we change this to "weTime"?

File: src/main/webapp/src/App-clear.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+            <p className="mb-2">The application is configured and ready to import our components.</p>
```
Comment (trinitydonohugh) [inline src/main/webapp/src/App-clear.js:14]: Ditto.

File: src/main/webapp/src/App-clear.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+            <MDBBtn href="https://mdbootstrap.com/docs/react/" target="blank" color="light-blue"><strong>Check out our docs!</strong></MDBBtn>
```
Comment (trinitydonohugh) [inline src/main/webapp/src/App-clear.js:15]: Ditto.

File: src/main/webapp/src/App.js [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
+            />
+            <MDBCollapse id='mainNavbarCollapse' isOpen={collapseID} navbar>
+              <MDBNavbarNav right>
+                <MDBNavItem>
+                  <MDBNavLink
+                    exact
+                    to='/'
+                    onClick={this.closeCollapse('mainNavbarCollapse')}
+                  >
+                    <strong>Home</strong>
```
Comment (trinitydonohugh) [inline src/main/webapp/src/App.js:64]: Can we only keep the nav items that we need: about and help. and have log in as a button?

File: src/main/webapp/src/Routes.js [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+
+import NavigationNavPage from './pages/NavigationNavPage';
+import FormsNavPage from './pages/FormsNavPage';
+import TablesNavPage from './pages/TablesNavPage';
+import AddonsNavPage from './pages/AddonsNavPage';
+import ModalsNavPage from './pages/ModalsNavPage';
+import AdvancedNavPage from './pages/AdvancedNavPage';
+import ComponentsNavPage from './pages/ComponentsNavPage';
+
+// FREE
```
Comment (trinitydonohugh) [inline src/main/webapp/src/Routes.js:14]: Can we only keep the components we need?

**Review by trinitydonohugh — CHANGES_REQUESTED** [review]
In general, can we only keep the pages and components we need for our website and remove the code that was specific to the starter pack? Thank you! :)

File: src/main/webapp/src/App-clear.js (hunk 6 above)
Comment (mbouadeus) [reply src/main/webapp/src/App-clear.js:13]: Yes, I'll add it in the next pull request.

File: src/main/webapp/src/App-clear.js (hunk 7 above)
Comment (mbouadeus) [reply src/main/webapp/src/App-clear.js:14]: Yes, I'll add it in the next pull request.

File: src/main/webapp/src/App-clear.js (hunk 8 above)
Comment (mbouadeus) [reply src/main/webapp/src/App-clear.js:15]: Yes, I'll add it in the next pull request.

File: src/main/webapp/src/App.js (hunk 9 above)
Comment (mbouadeus) [reply src/main/webapp/src/App.js:64]: Yes, I'll add it in the next pull request.

File: src/main/webapp/src/Routes.js (hunk 10 above)
Comment (mbouadeus) [reply src/main/webapp/src/Routes.js:14]: Yes, I'll add it in the next pull request.

**Review by bhargaviks — APPROVED** [review]

**Review by pengxianzhe — APPROVED** [review]
LGTM after all comments are resolved.

**Review by trinitydonohugh — APPROVED** [review]

**Pull request merged**